### PR TITLE
SKHandle Wrapper Class

### PR DESCRIPTION
**Pull request by rbharath**
`SKHandle` is a simple C++ class defined in HELib that defines a part of the secret key. Unlike many of the other HELib classes, it has the advantage of having a simple class structure without dependencies on the other classes in HELib.

This PR adds a simple python wrapper for `SKHandle` which allows for this class to be constructed from python. Also adds a simple python test for the class.

### CODE REVIEW

File: src/wrapper.cpp [hunk 1]
```diff
@@ -1,40 +1,52 @@
...
 PYBIND11_MODULE(hepy, m) {
+
+    // Wrapping for class SKHanlde in Ctxt.h
+    py::class_<SKHandle>(m, "SKHandle")
+      .def(py::init<long, long, long>())
+      .def("setBase", &SKHandle::setBase)
+      .def("isBase", &SKHandle::isBase)
+      .def("setOne", &SKHandle::setOne)
+      .def(py::self == py::self)
+      .def(py::self != py::self)
```
Comment (robrobbins) [inline src/wrapper.cpp]: wut?

Comment (rbharath) [reply src/wrapper.cpp]: Hah yes. This is the pybind11 magic setup for porting a C++ operator overload. `SKHandle::operator==` is ported into Python with `.def(py::self == py::self)`

**Comment by rbharath** [comment]
The MAC OSX build is failing with weird issues (the NTL headers aren't present for some reason). In the interest of development ease, I'm deactivating OSX support, since we're all developing on Linux. Would be good to figure out what's going on later.

**Comment by rbharath** [comment]
Ok, I think I know what's happening here. The `ntl` package is installed from Conda-forge into Anaconda, but the C++ compiler gets confused because the headers for this package aren't added into its path. I think I did this manually on my machine a while back, but this hasn't been done on Travis-CI. We need to set these environment variables accordingly.

https://stackoverflow.com/questions/44793143/build-program-using-library-and-headers-installed-in-a-conda-environment

File: src/wrapper.cpp (hunk 1 above)
Comment (robrobbins) [reply src/wrapper.cpp]: ic. so those 2 are wrappers for `==` operator and `!=` operator overloads

**Comment by rbharath** [comment]
Ok, this is passing now so going to go ahead and merge in. This is horrible and ugly though even by my forgiving standards. Will clean up in future PRs.

**Pull request merged**